packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -200,7 +201,7 @@ const Header: React.FC<Props> = ({
             aria-controls="sidebar"
             aria-haspopup="true"
             aria-expanded={sideBarOpen}
-            onClick={onMenuButtonClick}
+            onClick={() => onMenuButtonClick()}
```

Comment:
This is not needed... This is already done in the Layout (`onMenuButtonClick={() => setSideBarOpen(true)}`)

Reply:
I though so too! But during the process I found a bug that the `sideBarOpen` variable was true on page load. So wrapping `onMenuButtonClick` in an arrow function ensures that `onMenuButtonClick` is only executed when the `onClick` event occurs